source/agora/consensus/protocol/Nominator.d

```diff
@@ -455,6 +455,19 @@ extern(D):
             return;
         }
 
+        if (slot_idx > 1) // Genesis block is not signed
+        {
+            auto last_block = this.ledger.getLastBlock();
+            auto signed = last_block.header.validators;
+            if (signed.setCount <= signed.count() / 2)
```

Comment:
Nodes should also reject any nominations if they are missing signatures for the previous block, right?

Reply:
Yes will add check.

Reply:
Updated